system_tests/gitlab/test_gitlab_system.py

```diff
@@ -618,12 +619,15 @@ def test_assign_one_review(self, with_student_repos, tmpdir):
             expected_review_teams, single_group_assertion=group_assertion
         )
         assert_num_issues(STUDENT_TEAMS, [assignment_name], 1)
-        assert_issues_exist(
-            STUDENT_TEAMS,
-            [assignment_name],
-            _repobee.command.peer.DEFAULT_REVIEW_ISSUE,
-            expected_num_asignees=1,
-        )
+        for reviewed_team in STUDENT_TEAMS:
+            any_other_team_pattern = "|".join(
+                s.name for s in STUDENT_TEAMS if s != reviewed_team
+            )
+            assert_issue_title_match(
+                [reviewed_team],
+                [assignment_name],
+                rf"^Peer review \((?:{any_other_team_pattern})\)$",
+            )
```

Comment:
I would suggest to weaken the assertions here for the sake of simplicity. System tests are more about the bigger picture than minute details, so I think it's perfectly fine to have looser assertions than one normally would in a lower level test.

```suggestion
    assert_issue_title_match(STUDENT_TEAMS, [assignment_name], r"^Peer review.*$")
```